pkgs/development/python-modules/marshmallow-dataclass/default.nix

```diff
@@ -15,7 +14,7 @@ buildPythonPackage rec {
   version = "8.6.1";
   format = "setuptools";
 
-  disabled = pythonOlder "3.6";
+  disabled = pythonOlder "3.7";
```

Comment:
Could you please split this change into another commit?

Reply:
sure